### ISSUE TITLE
PP Filters: Allow users to specify the order filters appear in the list

### ISSUE THREAD

**Issue by john-thomas-dotcms** [dotCMS/core#18715]
As an administrator, I want to be able to specify the order in which PP filters appear in the list, in order to put the most commonly used filter(s) at the top of the list, without having to change the filter names.

### Describe the solution you'd like

Right now, PP filters are sorted in alphabetical order in the list.  This means that, if an admin wants one or more filters to appear at the top of the list, they must change the name of one or more filters to ensure the right ones appear at the top.

We should allow users to specify the order the filters appear in the list, overriding the default alphabetical sorting.

### Technical Approach

* Add an "order" property (integer) to the PP filter .yml file.
* If an "order" property exists in the file, set the order value to that number.
    * If no "order" property exists in the file, set the order value to some large number (so these get sorted last).
* Sort the filters in the list based on the order value.
    * If multiple filters have the same "order" value, sort them in alphabetical order at the appropriate point in the list.

So in summary, if the filter has no "order" property then treat is as having an order of 10000 or something like that, and then make the sort order basically `order, filterName`.

## Acceptance Criteria
1. Display the filters in proper order for combinations of filter files with and without the order property, and for filter files with the same order value.
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

**Comment by bryanboza** [comment]
Since this list is returned by an endpoint, we need to add some curl tests here.

**Comment by bryanboza** [comment]
Ok now if I do the call to the endpoint we added the sort property to the filters. But this order is not respected in the UI, we still ordering alphabetically. 
![image](https://user-images.githubusercontent.com/2641437/183151145-f8f47b30-81de-4e13-acaa-54640226eb52.png)

Here the UI:
![image](https://user-images.githubusercontent.com/2641437/183152308-f5ef3d65-2846-48db-a41c-ec962cb6e5a2.png)

**Comment by john-thomas-dotcms** [comment]
This card is specifically about the list the users see when doing a PP from the back-end UI.

So, can we please update the back-end UI so the list respects the order that's in the filters?
